### ISSUE TITLE
Maven Repository

### ISSUE THREAD

**Issue by samuel-pt** [zuora/codelibrary-java#4]
Can you push this to maven repository?

**Comment by hgschmie** [comment]
Yes. Let me work on that; it may not happen right away but I will try to get this done.

**Comment by woocash** [comment]
@samuel-pt & @hgschmie, we will once we're comfortable that this library is in a good shape. We anticipate fixing the packaging and some minor issues first. I expect that to happen this week, and we'd add it to Maven thereafter.

**Comment by samuel-pt** [comment]
@hgschmie & @woocash - Thanks for the update. Please update this ticket once this library has been pushed to maven repository

**Comment by Christian-Oette** [comment]
Any progress on the maven repository?

**Comment by earandes** [comment]
Do you know when will be released in maven repository?
